src/dialog-editor/services/dialogValidationService.ts

```diff
@@ -99,12 +100,28 @@ export default class DialogValidationService {
     return _.every(this.validators.fields, this.validate(field, label, local));
   }
 
+  /**
+   * Run validations across each dialog field elements and check for duplicate field names.
+   * @memberof DialogValidationService
+   * @function validateDialogFields
+   */
+  public validateDialogFields(field: any) {
+    this.dialogFields.push(field.name);
+    const isDuplicate = this.dialogFields.filter((item) => item === field.name).length > 1 ? true : false;
+    if (isDuplicate) {
+      this.invalid.message = `Please change duplicate field name "${field.name}"`;
```

Comment:
I think we should present this more directly.  Something like

```suggestion
      this.invalid.message = `Unable to save due to duplicate field name "${field.name}"`;
```

Reply:
Separate question - is it possible to add i18n support for this line?